homeassistant/components/squeezebox/media_player.py

```diff
@@ -65,52 +71,60 @@
     | SUPPORT_CLEAR_PLAYLIST
 )
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
-        vol.Optional(CONF_USERNAME): cv.string,
-    }
-)
-
-DATA_SQUEEZEBOX = "squeezebox"
-
-KNOWN_SERVERS = "squeezebox_known_servers"
-
+KNOWN_PLAYERS = "known_players"
+KNOWN_SERVERS = "known_servers"
+DISCOVERY_TASK = "discovery_task"
 ATTR_PARAMETERS = "parameters"
-
 ATTR_OTHER_PLAYER = "other_player"
 
 ATTR_TO_PROPERTY = [
     ATTR_QUERY_RESULT,
     ATTR_SYNC_GROUP,
 ]
 
+SQUEEZEBOX_MODE = {
+    "pause": STATE_PAUSED,
+    "play": STATE_PLAYING,
+    "stop": STATE_IDLE,
+}
+
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
-    """Set up the squeezebox platform."""
+    """Set up squeezebox platform from platform entry in configuration.yaml (deprecated)."""
+    _LOGGER.warning("Loading Squeezebox via platform config is deprecated.")
```

Comment:
Please use cv.deprecated in the platform schema instead.

Reply:
I removed the logger message, but I'm confused about the best way to use `cv.deprecated` here. The goal is to direct people to the config flow, not warn that certain options are not deprecated. I plan to support `configuration.yaml` for the foreseeable future, so maybe no warning is needed.